### DISCUSSION TITLE
Tailwindgenie.com new generative tool

### DISCUSSION THREAD

**Discussion by anuranjan87** [tailwindlabs/tailwindcss#14035] [Help]
feel free to give it a try, and give feedback

**Comment by navidemad** [comment]
The idea is cool, well done.
Some suggestions:

- Try to fix the flickering when rendering the code frame live.
- Rework the header to be consistent across all pages. Add your logo and navigation to it, and make it bigger.
- The layout change between / and /home is confusing; it feels like switching websites.
- Move your logo from the / page to the /home header and enlarge it.
- Redesign /home to improve its layout. There's too much empty space; a centered container would help. The current width feels too fluid.
- It took me a few seconds to realize there was an input field for my prompt. Initially, I thought it was just a showcase button.

Additionally, the $250/month pricing for the enterprise tier seems expensive for startups. Consider adding a pricing table with a freemium version, a premium version, and an enterprise version with a free license. You should reduce the initial price and gradually increase it based on the Monthly Recurring Revenue (MRR) of the project.

During the beta phase, you might want to hide the pricing when sharing the project.
